test/onnx/onnx_test_common.py

```diff
@@ -520,6 +519,7 @@ class DecorateMeta:
         test_behavior: The behavior of the test case. [skip or xfail]
         matcher: The matcher to apply to the test case.
         enabled_if: Whether to enable test behavior. Usually used on onnx/ort version control
+        torchmodeltype: The type of the torch model. Defaults to None.
```

Comment:
```suggestion
        torchmodeltype: The type of the torch model. Defaults to None.
```
At `test_fx_to_onnx_with_onnxruntime.py` this argument is named `model_type`. We should use the same name to make it uniform across the test files